examples/pyfunc/Pyfunc.ipynb

```diff
@@ -386,17 +388,53 @@
     "merlin.undeploy(v)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 4. Testing PyFunc Model Server Locally"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Up until this point, you have successfully built and run your PyFunc model on the Merlin server. To have faster development iteration, you can also simulate running the PyFunc model server in your local machine, by calling `merlin.run_pyfunc_model` function."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "merlin.run_pyfunc_model(\n",
```

Comment:
while running this, the kernel appear to get stuck, I had to change some configuration to allow scrollable logs on my IDE, to see that the pyfunc is actually running. 

im not sure if it is intentional that the below block is not a cell, i had to copy and curl in a separate terminal

Reply:
yeah, the kernel appear to get stuck because `run_pyfunc_model` is blocking function. We can modify it to be non-blocking, but it could be picked on separate PR.

i have also updated the next block that contains curl command to be executable instead of markdown.